### ISSUE TITLE
Feature - Select banner spacing background color

### ISSUE THREAD

**Issue by eladcandroid** [YoussefKababe/admob_flutter#31]
![Screen Shot 2019-05-29 at 14 27 57](https://user-images.githubusercontent.com/13297000/58553608-fe159300-821d-11e9-8e64-fabc7d709e37.png)

**Comment by kmcgill88** [comment]
@eladcandroid I agree with this feature request. Do you have any capacity to work on it?

**Comment by sumitgohil** [comment]
I am working on this. I have successfully set the background Color in Android. Looking forward to doing this in iOS too.

**Comment by kmcgill88** [comment]
@sumitgohil awesome to hear you picked this up! I'd be happy to review the PR when it's ready.

**Comment by kmcgill88** [comment]
@sumitgohil any update on this?

**Comment by EliasDeuss** [comment]
@sumitgohil are you still working on this issue?

**Comment by mahadydev** [comment]
this problem still persists. any update?

**Comment by mhassanist** [comment]
> I am working on this. I have successfully set the background Color in Android. Looking forward to doing this in iOS too.

May you share with us how ?

**Comment by armandojimenez** [comment]
any update or ways to do this?
